### PR TITLE
Issue #2566 Image: Exclude <none> tag from docker image list

### PR DESCRIPTION
**Pull request by praveenkumar**
How to test
--------------

```
$ minishift ssh
$ vi Dockerfile
FROM alpine:latest
MAINTAINER kumarpraveen.nitdgp@gmail.com
CMD pwd
$ docker build -t test:latest .
$ docker run test:latest

Edit the Dockerfile
$ vi Dockerfile
FROM alpine:latest
MAINTAINER kumarpraveen.nitdgp@gmail.com
CMD ls
$ docker build -t test:latest .
$ docker images
$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
test                latest              91b283f03e13        23 minutes ago      253 MB
<none>              <none>              6e5e13dbce1f        23 minutes ago      253 MB

$ exit

Host $ minishift image export --all <= without this PR you will get the error.
```

### CODE REVIEW

**Comment by praveenkumar** [comment]
CI build failure not because of any of current change but apline:latest image recently pushed to docker hub and not able to consume, same error I am getting in my local box. once it available we will retriger the test.

```
$ docker pull alpine:latest
Trying to pull repository docker.io/library/alpine ... 
latest: Pulling from docker.io/library/alpine
no supported platform found in manifest list

```

**Comment by praveenkumar** [comment]
retest this please.

**Comment by praveenkumar** [comment]
retest this please.

File: pkg/minishift/docker/image/oci_image_handler.go [hunk 1]
```diff
@@ -236,7 +236,11 @@ func (handler *OciImageHandler) GetDockerImages() (map[string]bool, error) {
 	}
 	defer session.Close()
 
-	cmd := "docker images --format '{{.Repository}}:{{.Tag}}'"
+	// We don't want to have an image which have <none> tag in our output list
+	// Also in case there is no image in docker daemon then exit code should be 0 instead 1
+	// and `|| true` is used.
+	// which refer to dangling image for docker.
+	cmd := "docker images --format '{{.Repository}}:{{.Tag}}' | grep -vw '<none>' || true"
```
Comment (LalatenduMohanty) [inline pkg/minishift/docker/image/oci_image_handler.go]: 👍

**Pull request merged**